### PR TITLE
Fix docstring in `exceptions.py`

### PR DESCRIPTION
**Pull request by urutva**
### Description

Fix docstring in `exceptions.py`

Signed-off-by: Devaraj Ranganna <devaraj.ranganna@arm.com>

### Test Coverage

- [ ]  This change is covered by existing or additional automated tests.
- [ ]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [x]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

File: news/20200706.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix docstring in exceptions.py
```
Comment (Patater) [inline news/20200706.doc:1]: Is this supposed to have a newline at the end or not?

Comment (urutva) [reply news/20200706.doc:1]: assert-news expect only one line per newsfile. If there is more than one line then it'll fail.

Comment (Patater) [reply news/20200706.doc:1]: It looks like there is a newline at the end of this file. Hopefully assert-news is OK with this, as vim usually ensures there is always a newline at the end of files when saving.

Comment (Patater) [reply news/20200706.doc:1]: Cool, looks like assert-news was OK with the newline at the end of the file. I'm glad. https://dev.azure.com/mbed-tools/mbed-build/_build/results?buildId=2791&view=logs&s=204402d8-1b03-5b49-8370-ed4d72352645

Comment (urutva) [reply news/20200706.doc:1]: Yeah, https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/mbed_tools_ci_scripts/assert_news.py#L47 ignores empty line :)

Comment (urutva) [reply news/20200706.doc:1]: Yeah, https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/mbed_tools_ci_scripts/assert_news.py#L47 ignores empty line :)

**Pull request merged**